dll/win32/shell32/CShellLink.cpp

```diff
@@ -2640,12 +2640,9 @@ HRESULT CShellLink::DoOpen(LPCMINVOKECOMMANDINFO lpici)
     sei.lpParameters = args;
     sei.lpClass = m_sLinkPath;
     sei.nShow = m_Header.nShowCommand;
+    if (lpici->nShow != SW_SHOWNORMAL && lpici->nShow != SW_SHOW)
+        sei.nShow = lpici->nShow; // Allow invoker to override .lnk show mode
     sei.lpDirectory = m_sWorkDir;
-    sei.lpVerb = L"open";
-
-    // HACK for ShellExecuteExW
-    if (m_sPath && wcsstr(m_sPath, L".cpl"))
-        sei.lpVerb = L"cplopen";
```

Comment:
If I understand correctly, this is now handled via the `SEE_MASK_IDLIST` ---> `SEE_MASK_INVOKEIDLIST` change (the crux of this PR)?

Reply:
Yes, SEE_MASK_INVOKEIDLIST means use IContextMenu and this becomes the same as double-clicking in the folder.